siliconcompiler/scheduler.py

```diff
@@ -112,11 +112,6 @@ def _deferstep(chip, step, index, status):
             sf.write('#!/bin/bash\n')
             sf.write(f'sc -cfg {shlex.quote(cfg_file)} -builddir {shlex.quote(buildir)} '
                      f'-arg_step {shlex.quote(step)} -arg_index {shlex.quote(index)}\n')
-            # In case of error(s) which prevents the SC build script from completing, ensure the
-            # file mutex for job completion is set in shared storage. This lockfile signals the
-            # server to mark the job done, without putting load on the cluster reporting/accounting
-            # system.
-            sf.write(f'touch {os.path.dirname(output_file)}/done')
```

Comment:
As far as I can tell this is not actually used anywhere.
This is only executed if the scheduler is slurm.
But the server uses `sacct` to check the status of the run in case slurm is used.
I think this was intended for remote runs that don't use slurm.
This might be something for @WRansohoff to chime in too as well.

Reply:
It's a file mutex which we have used in the past to mark the completion of jobs. Long-term we'd like to move this flag into less local storage, but I think Martin is correct that this particular one is no longer needed. The server-side logic handles the generation of these completion/error markers now.